components/site/header.tsx

```diff
@@ -44,12 +44,7 @@ const Header = () => {
         {category === 'momentum' ? (
           <Search indexName="next_momentum_documentation" />
         ) : (
-          <TextField
-            id="algolia-search"
-            label="Search"
-            prefix={<SearchIcon />}
-            placeholder="e.g. Getting Started"
-          />
+          <Search indexName="next_support_documentation" />
```

Comment:
Intentionally not checking the route because this needs to work at `/submit-a-ticket`